src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -151,6 +151,7 @@ public static class TextCatalog
         /// <format type="text/markdown">
         /// <![CDATA[
         /// [!code-csharp[ApplyWordEmbedding](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Text/ApplyCustomWordEmbedding.cs)]
+        /// [!code-fsharp[ApplyWordEmbedding](~/../docs/samples/docs/samples/Microsoft.ML.Samples.FSharp/Dynamic/Transforms/Text/ApplyCustomWordEmbedding.fs)]
```

Comment:
>ApplyCustomWordEmbedding [](start = 129, length = 24)

What is our intention here? Certainly we are not going to practically write equivalent F# samples for everything. Having one or two isolated ones is probably fine, but is there some plans for something broader? Not saying this is necessarily the place for it, just wondering if we have a broader strategy.